docker-compose.yml

```diff
@@ -9,6 +9,7 @@ web:
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+  command: bundle exec unicorn -l 0.0.0.0:3000
```

Comment:
I think the problem might be backwards from what you're thinking. I was finding that `docker-compose` wasn't honoring my `Dockerfile` (and at its root, that problem might have just been needing a restart of the VM or something). I could put anything in my `Dockerfile`, and it wasn't being respected by `docker-compose`.

If EB and CCI aren't using `docker-compose`, this likely wouldn't be a problem for them at all, no?